packages/gatsby-theme-newrelic/gatsby/on-route-update.js

```diff
@@ -1,7 +1,17 @@
 import trackViaTessen from '../src/utils/page-tracking/tessen';
+import getLocale from '../src/utils/getLocale';
 
 const onRouteUpdate = ({ location, prevLocation }, themeOptions) => {
   trackViaTessen({ location, prevLocation }, themeOptions);
+
+  if (themeOptions.newrelic && window.newrelic) {
+    const { i18n } = themeOptions;
```

Comment:
```suggestion
    const { i18n: i18nConfig } = themeOptions;
```

Should this get the wrapped i18n config using the `withDefaults` helper? In the event `i18n` isn't configured, this value would be `undefined`, which may break the `getLocale` function. Also, could we name this `i18nConfig` just so we don't confuse it with the `i18n` instance? You can see an example here: https://github.com/newrelic/gatsby-theme-newrelic/blob/develop/packages/gatsby-theme-newrelic/gatsby/wrap-page-element.js#L15

In fact, it might even be better just to pass the full `themeOptions` to the `getLocale` function and have that function wrap using `withDefaults` so that you don't have to repeat it between the `onRouteUpdate` and `onClientEntry` hooks.

Reply:
Good idea! I will make that change